extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -153,9 +153,11 @@ const renderOutlinkUI = (pageEl, attachmentEl) => {
     anchorChild?.textContent ||
     attachmentEl.getAttribute('cta-text') ||
     attachmentEl.getAttribute('data-cta-text');
-  const openLabel = openLabelAttr
-    ? openLabelAttr.trim()
-    : localize(pageEl, LocalizedStringId.AMP_STORY_PAGE_ATTACHMENT_OPEN_LABEL);
+  // TODO(#36724): Clean this up once the "Read more" CTA has been translated.
+  const fallbackCta =
+    localize(pageEl, LocalizedStringId.AMP_STORY_READ_MORE) ??
```

Comment:
I think this will always return the default string in english for `AMP_STORY_READ_MORE` so there's no point in setting the fallback to `...PAGE_ATTACHMENT_OPEN_LABEL`. The `localize` function checks the document's language and the default language before returning null, so it wouldn't happen at all. If you want to do this, you'd need to directly check `doc.lang == "en" ? LocalizedStringId.AMP_STORY_READ_MORE : LocalizedStringId.AMP_STORY_PAGE_ATTACHMENT_OPEN_LABEL`

Reply:
Thanks for the info! I've reverted the changes in this file because, on second thought, I would rather wait until the new string has been fully translated so that I can update the CTAs and the [documentation](https://amp.dev/documentation/components/amp-story-page-attachment/#cta-text-(optional)) simultaneously. Since translations can take a while to be completed, I'm hesitant to allow the CTAs and documentation to diverge  for potentially several weeks